doc/conf.py

```diff
@@ -88,6 +89,11 @@
 # The master toctree document.
 master_doc = 'index'
 
+# sphinxcontrib-bibtex
+bibtex_bibfiles = ['./references.bib']
+bibtex_style = 'unsrt'
```

Comment:
```suggestion
bibtex_style = 'unsrt'
bibtex_reference_style = 'author_year'
```

Reply:
Thanks @emdupre ! :+1: 
It looks like `bibtex_style` and `bibtex_reference_style` are not taken into account. I tried all possible values locally and nothing seems to change the rendering... :confused: 
Setting them to invalid values does raise an error though.
I'm probably missing something here...

Reply:
It looks like `bibtex_reference_style` currently only works with `:cite:t:` and `:cite:p:`, see https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/240 although they are planning to support footcite at some point.

I'll keep an eye on this, but I think we can keep the current formatting while waiting for author-year style to become available. WDYT?

Reply:
I am fine with the plan, if nobody has a solution to author-year display. You may want to contribute upstream to sphinx ;-)

Reply:
Sounds good to me ! Thanks, @NicolasGensollen !